src/docs/data-template-dir.md

```diff
@@ -33,7 +33,7 @@ For example, consider a template located at `posts/subdir/my-first-blog-post.md`
 
 ### Apply a default layout to multiple templates
 
-{% codetitle "posts/post.json" %}
+{% codetitle "posts/posts.json" %}
```

Comment:
I think L42 below would need to change also: https://www.11ty.dev/docs/data-template-dir/#apply-a-default-layout-to-multiple-templates

> _"Using the above in `posts/post.json` will configure a layout for all of the templates inside of `posts/*`."_

Reply:
done, commits squashed.